mozilla/kuma
# bug 957802 - Add dennis, make targets for locale, local linting

dennis is used to check locale files for common errors.

Added some make targets for locale work:

* make localetest - Run dennis on locales
* make localeextract - Extract translatable strings
* make localecompile - Compile translations for local use
* make localerefresh - Run all the steps to update locales, except for commit

Updated the localization documentation for these new targets.

The only string update was the addition of "B2G OS", which I don't believe should be localized, so I'm not including a string update this week.
